src/client/scripts/game/chess/engine.js

```diff
@@ -0,0 +1,50 @@
+
+/**
+ * This script runs the chess engine for enginegames
```

Comment:
Could be added that should should be modular and able to be switches out with other engine scripts?